src/redisenterprise/azext_redisenterprise/generated/_params.py

```diff
@@ -132,11 +141,17 @@ def load_arguments(self, _):
                    'RedisEnterprise cluster.', id_part='name')
         c.argument('sas_uri', type=str, help='SAS URI for the target directory to export to')
 
+    with self.argument_context('redisenterprise database force-unlink') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('cluster_name', options_list=['--cluster-name', '--name', '-n'], type=str, help='The name of the '
+                   'RedisEnterprise cluster.', id_part='name')
+        c.argument('unlink_ids', nargs='+', help='The resource IDs of the database resources to be unlinked.')
+
     with self.argument_context('redisenterprise database import') as c:
         c.argument('resource_group_name', resource_group_name_type)
         c.argument('cluster_name', options_list=['--cluster-name', '--name', '-n'], type=str, help='The name of the '
                    'RedisEnterprise cluster.', id_part='name')
-        c.argument('sas_uri', type=str, help='SAS URI for the target blob to import from')
+        c.argument('sas_uris', nargs='+', help='SAS URIs for the target blobs to import from')
```

Comment:
Renaming this parameter is a breaking change, can you double confirm?

Reply:
Yes we intend to rename this param